src/taipy/core/taipy.py

```diff
@@ -577,6 +584,23 @@ def create_scenario(
     return _ScenarioManagerFactory._build_manager()._create(config, creation_date, name)
 
 
+def create_global_data_node(config: DataNodeConfig) -> DataNode:
+    """Create and return a new data node from a data node configuration with GLOBAL scope.
+
+    Parameters:
+        config (DataNodeConfig^): The data node configuration. The configuration must have GLOBAL scope.
+    Returns:
+        The new global data node.
+    Raises:
+        DataNodeConfigIsNotGlobal^: If the data node configuration does not have GLOBAL scope.
+    """
+    # Check if the data node config has GLOBAL scope
+    if config.scope is not Scope.GLOBAL:
+        raise DataNodeConfigIsNotGlobal(config.id)  # type: ignore
+
+    return _DataManagerFactory._build_manager()._create_and_set(config, None, None)
```

Comment:
Instead of using `_create_and_set` should we use `_bulk_get_or_create` method?
Because we don't want to create the data node twice if it already exists.

```suggestion
    return _DataManagerFactory._build_manager()._bulk_get_or_create([config])
```

Reply:
You are right, I will update this and add a unittest